### PR TITLE
Completed concept entry for Postgresql operators #4764

### LINKED ISSUE #4649: [Concept Entry] Postgresql operators
### Reason/inspiration (optional)

We would like a new entry on the `Operators` concept under PostgreSQL. The entry should go in a new file under `docs/content/postgresql/concepts/operators/operators.md`.
 
 The entry should include:
 
 - An introductory paragraph that describes the concept
 - A ‘Syntax’ section that includes the syntax for the concept (if applicable)
 - An ‘Example’ section that shows the concept in use
 
 Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by nbarnhouse**
### Description

Completed concept entry for Postgresql operators

### Issue Solved

Closes #4649

### Type of Change

- Adding a new entry

### Checklist

- [x ] All writings are my own.
- [ x] My entry follows the Codecademy Docs style guide.
- [ x] My changes generate no new warnings.
- [ x] I have performed a self-review of my own writing and code.
- [ x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by nbarnhouse** [comment]
Submitted this to fix error in https://github.com/Codecademy/docs/pull/4764. ++ @cigar-galaxy82

File: content/postgresql/concepts/operator-invocations/operator-invocations.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data'
+  - 'Operators'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/data-science-foundations'
```
Comment (cigar-galaxy82) [inline content/postgresql/concepts/operator-invocations/operator-invocations.md:13]:
```suggestion
  - 'learn-sql'
  - 'paths/data-science-foundations'
  - 'paths/analyze-data-with-sql'
  - 'paths/design-databases-with-postgresql'
```

File: content/postgresql/concepts/operator-invocations/operator-invocations.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+SELECT * from table_name
+WHERE column1 >= 10 AND  column1 <= 1000;
+
+-- Or
+SELECT * from table_name
+WHERE column1 = 'North' OR 'WEST';
+
+-- Not
+SELECT * from table_name
+WHERE NOT column1 = 'SOUTH';
```
Comment (cigar-galaxy82) [inline content/postgresql/concepts/operator-invocations/operator-invocations.md:55]:
```suggestion
// AND
SELECT * FROM table_name
WHERE column1 >= 10 AND  column1 <= 1000;

// OR
SELECT * FROM table_name
WHERE column1 = 'North' OR 'WEST';

// NOT
SELECT * FROM table_name
WHERE NOT column1 = 'SOUTH';
```

File: content/postgresql/concepts/operator-invocations/operator-invocations.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+SELECT * from table_name
+WHERE column1 = column2;
+
+-- Greater Than (or Equal To)
+SELECT * from table_name
+WHERE column1 > 10;
+
+-- Less Than (or Equal To)
+SELECT * from table_name
+WHERE column1 < 1000;
```
Comment (cigar-galaxy82) [inline content/postgresql/concepts/operator-invocations/operator-invocations.md:39]:
```suggestion
// Equal to
SELECT * FROM table_name
WHERE column1 = 'North';

// Not Equal to
SELECT * FROM table_name
WHERE column1 != column2;

// Greater than
SELECT * FROM table_name
WHERE column1 > 10;

// Less than
SELECT * FROM table_name
WHERE column1 < 1000;
```

File: content/postgresql/concepts/operator-invocations/operator-invocations.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+| ---- | -------- | ----------- | ---------- | ------ |
+| 01   | Niko     | monkey      | female     | 3      |
+| 02   | Frank    | giraffe     | male       | 4      |
+| 03   | Lyn      | lion        | female     | 6      |
+| 04   | Tom      | elephant    | male       | 7      |
+
+
+```sql
+SELECT * from animals
+WHERE gender = 'female' AND age >= 4;
```
Comment (cigar-galaxy82) [inline content/postgresql/concepts/operator-invocations/operator-invocations.md:73]:
```suggestion
SELECT * FROM animals WHERE gender = 'female' AND age >= 4;
```

File: content/postgresql/concepts/operator-invocations/operator-invocations.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+| 03   | Lyn      | lion        | female     | 6      |
+
+
+## Example 2
+
+Using that same table, here is another operator invocation example.
+
+```sql
+SELECT * from animals
+WHERE NOT species = 'lion';
```
Comment (cigar-galaxy82) [inline content/postgresql/concepts/operator-invocations/operator-invocations.md:89]:
```suggestion
SELECT * FROM animals WHERE NOT species = 'lion';
```

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]
Hi, I have suggested a few changes please modify

**Comment by cigar-galaxy82** [comment]
Hi @nbarnhouse 

Thank you for contributing to Codecademy docs. I have suggested a few changes please modify

**Comment by nbarnhouse** [comment]
Thank you @cigar-galaxy82. I have made updates based on your comments.

**Review by cigar-galaxy82 — APPROVED** [review]

**Review by Sriparno08 — APPROVED** [review]
Looks good, @nbarnhouse! 👍

**Pull request merged**